common/app/conf/switches/IdentitySwitches.scala

```diff
@@ -15,33 +15,13 @@ trait IdentitySwitches {
     exposeClientSide = true
   )
 
-  val IdentityGdprMarketingConsentSwitch = Switch(
```

Comment:
I don't think this approach is necessarily bad, but I would've just removed this switch as well and left `IdentityPointToConsentJourneyPage`?